app/frontends/ogc-coverages/get-coverage-rangeset.ts

```diff
@@ -70,6 +70,11 @@ export default function getCoverageRangeset(
     if (bbox) {
       operation.boundingRectangle = bbox;
     }
+    const point = parsePointParam(query.point);
+    if (point) {
+      if (bbox) throw new RequestValidationError('bounding_box and point query parameters should not co-exist');
```

Comment:
What about point and shapefile? Should we throw a validation error for that as well?

Reply:
Done.